test/scripts/base-host-check.sh

```diff
@@ -170,6 +170,21 @@ check_services_disabled() {
     done
 }
 
+check_services_masked() {
+    echo "📗 Checking masked services"
+
+    services_expected=$(jq -rc '.blueprint.customizations.services.masked[]' "${config}")
+
+    echo "$services_expected" | while read -r service_expected; do
+        states=$(systemctl list-unit-files --state=masked)
+        if echo "${states}" | grep -q "${service_expected}"; then
+            echo "Service was masked service=${service_expected}"
+        else
+            echo "❌ Service was not masked service=${service_expected} output=${states}"
+        fi
+    done
```

Comment:
ditto: shouldn't the check `exit 1` in case there are any failures?

Reply:
Done.